.vscode/cspell.json

```diff
@@ -70,5 +70,8 @@
     "rust",
     "softwareTerms",
     "en_US"
+  ],
+  "enableFiletypes": [
+    "slice"
   ]
```

Comment:
This seems unrelated to the PR, is it required?

Reply:
You got here before I edited the comment!

> This PR updates the version, and pulls in 2 commits that are on main, which would be nice to have on 0.3.x as well:
fixes to the language extension settings.

Reply:
I was going to rebase this PR to pull them along, but keep the commits seperate.